### PR TITLE
Install devise, add user

### PR DESCRIPTION
**Pull request by PauladO**
Install devise, add user #16

### CODE REVIEW

**Comment by Cesurhan** [comment]
Confirmed.

File: app/models/view.rb [hunk 1]
```diff
@@ -0,0 +1,6 @@
+class View < ApplicationRecord
```
Comment (arnoFleming) [inline app/models/view.rb]: What is the purpose of this model?

File: app/views/layouts/application.html.erb [hunk 2]
```diff
@@ -9,6 +9,9 @@
   </head>
 
   <body>
-    <%= yield %>
+  <p class="notice"><%= notice %></p>
```
Comment (arnoFleming) [inline app/views/layouts/application.html.erb]: please reindent

File: config/routes.rb [hunk 3]
```diff
@@ -1,6 +1,7 @@
 Rails.application.routes.draw do
 
-
+  devise_for :views
```
Comment (arnoFleming) [inline config/routes.rb]: do we need this?

File: spec/models/user_spec.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe User, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (arnoFleming) [inline spec/models/user_spec.rb]: do this

File: spec/models/view_spec.rb [hunk 5]
```diff
@@ -0,0 +1,5 @@
+require 'rails_helper'
+
+RSpec.describe View, type: :model do
+  pending "add some examples to (or delete) #{__FILE__}"
```
Comment (arnoFleming) [inline spec/models/view_spec.rb]: idem

File: spec/factories/views.rb [hunk 6]
```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
```
Comment (arnoFleming) [inline spec/factories/views.rb]: remove or use this factory

File: spec/factories/users.rb [hunk 7]
```diff
@@ -0,0 +1,5 @@
+FactoryGirl.define do
+  factory :user do
```
Comment (arnoFleming) [inline spec/factories/users.rb]: remove or use this factory

**Comment by arnoFleming** [comment]
~GLBT~ LGTM

**Comment by arnoFleming** [comment]
meh, please merge in master

**Pull request merged**